mozilla/fx-private-relay
# Promo-blocking UI for free users

~~(Still set to draft because this feature includes a link to the waitlist, which hasn't been merged yet: https://github.com/mozilla/fx-private-relay/pull/1846.)~~

# New feature description

This implements the promo-blocking slider for free users instead of the toggle, with the promo-blocking step simply explaining the feature and its availability for Premium users. It also adds a toggle to enable promo-blocking when creating a custom alias.

After this is merged, we can also uncomment a line in this PR: https://github.com/mozilla/fx-private-relay/pull/1672/files#r833157494

# Screenshot (if applicable)

[Free slider user](https://deploy-preview-1673--fx-relay-demo.netlify.app/?mockId=empty) (create an alias first):

https://user-images.githubusercontent.com/4251/161987266-cbc9ca20-7005-4de1-99e0-18f88bba7c72.mp4

[Checkbox on alias creation](https://deploy-preview-1673--fx-relay-demo.netlify.app/?mockId=some):

https://user-images.githubusercontent.com/4251/161987210-f704acd2-30e9-40c3-b88c-18d70ab539a5.mp4

# How to test

All alias cards should show the slider. Free users should see the explanation about promo-blocking and Premium, Premium users should just be able to set it. Free users in a country without Premium should be linked to the waitlist page (which hasn't been implemented yet).

# Checklist
